### ISSUE TITLE
Running on folder is not working

### ISSUE THREAD

**Issue by renatoliveira** [renatoliveira/apexlint#1]
Running in a single file the linting works, but when specifying a folder it does not.

```
📂 - Running on folder src/classes...
✅ - No errors found.
```

```
📄 - Running on file src/classes/OpportunityWS.cls...
❌ - 29: Line exceeds the limit of 120 characters.
❌ - 117: Line exceeds the limit of 120 characters.
❌ - 127: SOQL query with more than one condition and/or field being searched should be splitted into multiple lines.
❌ - 138: Line exceeds the limit of 120 characters.
```
